person.rb

```diff
@@ -1,9 +1,12 @@
-class Person
+require_relative './decorator'
+
+class Person < Nameable
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
```

Comment:
- [ ] **[OPTIONAL]** Adding super call here is not necessary since `Nameable` class does not have a constructor.

Reply:
Oh ok.